hl7apy/__init__.py

```diff
@@ -308,7 +311,7 @@ def find_reference(name, element_types, version):
 
 def load_message_profile(path):
     with open(path) as f:
```

Comment:
Most of the tests fails because of this line.
It raises `TypeError:  'str' does not support the buffer interface`.
It should be sufficient to change it to `with open(path, 'rb') as f:` to make it work